src/components/home/vacations-card.tsx

```diff
@@ -48,10 +50,10 @@ const VacationsCard = () => {
     adminMode ? allVacationRequestStatusesAtom : vacationRequestStatusesAtom
   );
   const [loading, setLoading] = useState(false);
-  const [persons] = useAtom(personsAtom);
-  const loggedInPerson = persons.find(
-    (person: Person) =>
-      person.id === config.person.forecastUserIdOverride || person.keycloakId === userProfile?.id
+  const [users] = useAtom(usersAtom);
+  const loggedInPerson = users.find(
+    (person: User) =>
+      person.forecastId === config.person.forecastUserIdOverride || person.id === userProfile?.id
```

Comment:
```suggestion
      user.id === userProfile?.id
```

Reply:
LoggedInPerson should also be renamed to loggedInUser. Again apply to all files changes in this PR